### PR TITLE
Remove Paella 6 with OC 16

### PR DESCRIPTION
**Pull request by gregorydlogan**
This PR removes Paella 6 from Opencast, per the proposal on the users list

### Your pull request should…

* [x] have a concise title
* [ ] ~[close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists~
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] ~include migration scripts and documentation, if appropriate~
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: .github/workflows/auto-update-npm-libraries.yml [hunk 1]
```diff
@@ -35,15 +35,6 @@ jobs:
           npm run eslint
           git add package.json package-lock.json
 
-      - name: update paella-player test libraries
```
Comment (gregorydlogan) [inline .github/workflows/auto-update-npm-libraries.yml:38]: There's no equivalent stanza here for paella 7.  Should there be?

**Review by Arnei — CHANGES_REQUESTED** [review]
While this does remove Paella6, it does not remove Paella6 from the docs :D We should probably do that so as to avoid confusion among our adopters.

**Comment by Arnei** [comment]
Also you did not catch all entries in the ´feature.xml` I think?

**Comment by Arnei** [comment]
Then there is also the config option to set either paella6 or 7, which is now obsolete? Even if we want to keep it around for the feature, we should not mention paella6 as a valid option.

https://github.com/opencast/opencast/blob/5ddc5ad23d80d86bf9f6ca1764318e19f03b17b2/etc/org.opencastproject.organization-mh_default_org.cfg#L185

**Comment by Arnei** [comment]
These rules can go as well, in theory. Maybe some adopters are doing redirect shenanigans that are relying on these rules.

https://github.com/opencast/opencast/blob/5ddc5ad23d80d86bf9f6ca1764318e19f03b17b2/etc/security/mh_default_org.xml#L290

**Comment by Arnei** [comment]
Otherwise I did some minimal testing to confirm that Paella7 still works (so we don't suddenly have zero working players).

**Comment by gregorydlogan** [comment]
Think I got it all.  The only thing I didn't see was more references in the feature.xml file - that's just paella 7 as far as I can tell.

File: .github/workflows/auto-update-npm-libraries.yml (hunk 1 above)
Comment (lkiesow) [reply .github/workflows/auto-update-npm-libraries.yml:38]: There could be, but Dependabot will pick that up.
Would reduce the Dependabot pull requests though.

**Pull request merged**